conjur/cli.py

```diff
@@ -75,12 +75,16 @@ def main_epilog():
 '''
 
     @staticmethod
-    def command_epilog(*args):
+    def command_epilog(example, has_subcommand=False):
         """
         This method builds the footer for each command help screen.
         """
-        return '''Examples:
-    {}'''.format(*args)
+        refer_to_help = "Refer to the specific subcommand help for more details."
+        if has_subcommand:
+            return f'''Examples: \n{refer_to_help}
```

Comment:
IS the newline required? Can you do the following?
```suggestion
            return f'''Examples:
            {refer_to_help}
```

Reply:
it will turn into
```
       if has_subcommand:
            return f'''Examples:
{refer_to_help}
```
because of the sensitive formatting

Reply:
still looks better IMO

Reply:
updated